shared/middleware/assets_api.rb

```diff
@@ -35,7 +35,16 @@ class AssetsApi < Sinatra::Base
     s3 = Aws::S3::Client.new(region: 'us-east-1')
     s3.list_objects(bucket:CDO.assets_s3_bucket, prefix:prefix).contents.map do |fileinfo|
       filename = %r{#{prefix}/(.+)$}.match(fileinfo.key)[1]
-      {filename:filename, size:fileinfo.size}
+      mime_type = Sinatra::Base.mime_type(filename.split('.').last)
+      case mime_type.split('/').first
+        when 'image'
+          category = 'image'
+        when 'audio'
+          category = 'sound'
+        else
+          halt(500, 'Unexpected mime type: ' + mime_type)
```

Comment:
We may not want to `500` on a client error.  `415 Unsupported Media Type` could be a better fit.